### PR TITLE
Rubocop fixes

### PR DESCRIPTION
**Pull request by adamruzicka**
I enabled cops for things which caught my eye when reading through Dynflow/dynflow#209 and fixed the code to make rubocop happy. I also added some minor style fixes where it made sense to me but without enforcing them.

I don't want to force my personal opinions about how the code should look onto the project so please feel free to pick from the commits.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1 +1,35 @@
...
+
+# Cop supports --auto-correct.
+# Configuration parameters: AllowForAlignment, ForceEqualSignAlignment.
+Style/ExtraSpacing:
+  Enabled: true
+
+# Cop supports --auto-correct.
+# Configuration parameters: IndentationWidth.
+Style/IndentAssignment:
+  Enabled: true
```
Comment (iNecas) [inline .rubocop.yml]: I would like to get to non-inlined versions eventually, I don't like much the fact that when you change one assignment, the whole block needs to get updated

**Review by iNecas — CHANGES_REQUESTED** [review]
One cop I'm not in favour, but big plus for doing this!

**Comment by adamruzicka** [comment]
@iNecas reverted that one back

**Comment by iNecas** [comment]
Thanks @adamruzicka !!!

File: lib/dynflow/director/sequence_cursor.rb [hunk 2]
```diff
@@ -65,23 +64,18 @@ def done_here?
...
         end
       end
 
       def add_todo(flow)
         case flow
         when Flows::Sequence
-          @todo << SequenceCursor.new(@flow_manager, flow, self).tap do |cursor|
-            cursor.move
-          end
+          @todo << SequenceCursor.new(@flow_manager, flow, self).tap(&:move)
```
Comment (iNecas) [inline lib/dynflow/director/sequence_cursor.rb:78]: Funny thing, this two things are not the same, as the `&:` variant has issues wiht protected method. I will revert this commit in my PR, as I don't want to risk to hit this unexpected behaviour changes in alredy pretty big PR

```
SequenceCursor.new(@flow_manager, flow, self).tap(&:move)
# NoMethodError: protected method `move' called for #<Dynflow::Director::SequenceCursor:0x0055555fbc9b70>

```

Comment (adamruzicka) [reply lib/dynflow/director/sequence_cursor.rb:78]: That's unexpected. Sorry about that. I now went through all the changes introduced by the Style/SymbolProc cop and this issue shouldn't show up anywhere else, but better be safe than sorry. Let's not risk it

**Pull request merged**